requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.8.2
+-e git+https://github.com/coolharsh55/python-opensubtitles@f37074796ee72c7efe1aa3ec2d534b4299b8fd88#egg=python_opensubtitles-master
```

Comment:
It installs via pip just fine.
The problem arises in pip freeze, where it gives an error on
python-opensubtitles’ svn repo.
```Warning: cannot find svn location for ...```

See this link on SO about the problem ->
http://stackoverflow.com/questions/8860731/pip-freeze-requirements-txt-error

The bug/issue in pip is referenced here ->
pip error #287 https://github.com/pypa/pip/issues/287

Since the repo code is on Github (git) I think removing the svn linking
is safe (at least in development versions), since this issue also has 
been reported when using mercurial.

If you think there needs to be more resolution (changes), please let me
know, since I’d be willing to trying to fix this.


--  
Harshvardhan Pandit
MSc Computer Science
http://harshp.com

-----Original Message-----
From: Alexandre González <notifications@github.com>
Reply: agonzalezro/python-opensubtitles <reply@reply.github.com>>
Date: 11 August 2015 at 16:12:04
To: agonzalezro/python-opensubtitles <python-opensubtitles@noreply.github.com>>
Cc: Harshvardhan Pandit <me@harshp.com>>
Subject:  Re: [python-opensubtitles] fix pip freeze svn requirements error (#8)

> > @@ -0,0 +1,3 @@
> > +Django==1.8.2
> > +-e git+https://github.com/coolharsh55/python-opensubtitles@f37074796ee72c7efe1aa3ec2d534b4299b8fd88#egg=python_opensubtitles-master  
>  
> To be fair, I don't completely understand the problem but this doesn't look completely  
> right.
>  
> Last time I checked I could install this with `pip`. But I didn't do it for a long time.
>  
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/agonzalezro/python-opensubtitles/pull/8/files#r36731662